### PR TITLE
My fifth and sixth(5 and 6) homeworks were added.

### PR DESCRIPTION
**Pull request by mariianakurta**
My fifth homework project was added.

### CODE REVIEW

File: ALevelHomework5/CodeApp.deps.json
Comment (petrochernykh) [inline ALevelHomework5/CodeApp.deps.json:1]: I see that you put here .cs files, but I cannot see .sln file, please make sure next time to put it here as well, as this is an important part of the project

File: ALevelHomework5/Scripts/Class1.cs [hunk 1]
```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+
+public enum LogType
```
Comment (petrochernykh) [inline ALevelHomework5/Scripts/Class1.cs:5]: It's better to move this enum to a separate file, but good that you decided to use it

File: ALevelHomework5/Scripts/Class2.cs [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    {
+        return Logging(true, LogType.Warning, "Skipped logic in method");
+    }
+
+    public Result ErrorMethod()
+    {
+        return Logging(false, LogType.Error, "I broke a logic");
+    }
+
+    private Result Logging(bool status, LogType logType, string message)
```
Comment (petrochernykh) [inline ALevelHomework5/Scripts/Class2.cs:24]: Looks good

File: ALevelHomework5/Scripts/Class3.cs [hunk 3]
```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+
+public class Result
+{
+    public bool Condition { get; set; }
+    public string Error2 { get; set; }
```
Comment (petrochernykh) [inline ALevelHomework5/Scripts/Class3.cs:8]: It seems that this is a typo

**Review by mariianakurta — COMMENTED** [review]
6th homework project was added.

File: ALevelHomework6.0/ALevelHomework6.0/Program.cs [hunk 4]
```diff
@@ -0,0 +1,119 @@
+﻿using System;
+
+internal class Products
```
Comment (petrochernykh) [inline ALevelHomework6.0/ALevelHomework6.0/Program.cs:3]: We need to separate this bg file and create one file per one class

File: ALevelHomework6.0/ALevelHomework6.0/Program.cs [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+            cart.Components[itemCount] = product;
+            itemCount++;
+        }
+        else
+        {
+            Console.WriteLine("Your cart is overflowing with animals");
+        }
+    }
+
+    public ShopCart GetCart()
```
Comment (petrochernykh) [inline ALevelHomework6.0/ALevelHomework6.0/Program.cs:61]: It's better to user get;set;

File: ALevelHomework6.0/ALevelHomework6.0/Program.cs [hunk 6]
```diff
@@ -0,0 +1,119 @@
...
+    private ShopCart cart;
+
+    public ShopServices(int cartCapacity)
+    {
+        cart = new ShopCart(cartCapacity);
+    }
+
+    private int itemCount = 0;
+
+    public void AddAnimal(Products product)
```
Comment (petrochernykh) [inline ALevelHomework6.0/ALevelHomework6.0/Program.cs:48]: AddAnimal?

File: ALevelHomework6/ALevelHomework6/Program.cs
Comment (petrochernykh) [inline ALevelHomework6/ALevelHomework6/Program.cs:1]: Why you have two files named ALevelHomework6.0 and ALevelHomework6?

File: ALevelHomework6.0/ALevelHomework6.0/Program.cs [hunk 7]
```diff
@@ -0,0 +1,119 @@
...
+                    {
+                        Console.WriteLine("This number is not in the list.");
+                    }
+                }
+            }
+            else
+            {
+                Console.WriteLine("Invalid input. Enter a number from the list.");
+            }
+        }
```
Comment (petrochernykh) [inline ALevelHomework6.0/ALevelHomework6.0/Program.cs:117]: It's better to move this to a separate method

File: ALevelHomework6.0/ALevelHomework6.0/Program.cs [hunk 8]
```diff
@@ -0,0 +1,119 @@
...
+
+internal class ShopCart
+{
+    public Products[] Components { get; set; }
+
+    public ShopCart(int ability)
+    {
+        Components = new Products[ability];
+    }
+}
```
Comment (petrochernykh) [inline ALevelHomework6.0/ALevelHomework6.0/Program.cs:18]: I don't understand this class, we need at least properties of  id, sum, date so on

**Pull request merged**